cmd/chart.go

```diff
@@ -89,12 +89,16 @@ func moveChart(cmd *cobra.Command, args []string) error {
 		mover.WithRetries(retries), mover.WithLogger(cmd),
 	)
 	if err != nil {
-		if err == mover.ErrImageHintsMissing {
+		var loadingError *mover.ChartLoadingError
+		if errors.As(err, &loadingError) {
```

Comment:
This is interesting, just read a little bit more about the use of `As`. 

Do you really need to do this unwrap, if it's a type checking on the top level error type, can't you just do the a comparison similar to the other errors?

```
if err == mover.ChartLoadingError {
  return err
}  
```